chargeio/services/PaymentMethods/TokenReferenceOptions.cs

```diff
@@ -1,14 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Newtonsoft.Json;
-
-namespace ChargeIO
-{
-    [JsonConverter(typeof(TokenReferenceConverter))]
-    public class TokenReferenceOptions : IPaymentMethod
-    {
-        public string TokenId { get; set; }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace chargeio
+{
+    [JsonConverter(typeof(TokenReferenceConverter))]
+    public class TokenReferenceOptions

```

Comment:
Similar to BankOptions, the IPaymentMethod marker interface is no longer implemented?